http-client-core/src/main/java/io/micronaut/http/client/interceptor/HttpClientIntroductionAdvice.java

```diff
@@ -400,6 +400,18 @@ private RequestBinderResult bindRequest(MethodInvocationContext<Object, Object>
         // The original query can be added by getting it from the request.getUri() and appending
         request.uri(URI.create(appendQuery(uri, uriContext.getQueryParameters())));
 
+        final MediaType[] acceptTypes;
+        Collection<MediaType> accept = request.accept();
+        if (accept.isEmpty()) {
```

Comment:
should we use `CollectionUtils::isEmpty` to protect against a NPE.

Reply:
I don't think we need to in this case - the contract of HttpRequest.accept() is to always return "A list of zero or many MediaType instances"